R/RcppSMC.R

```diff
@@ -0,0 +1,38 @@
+#' A package for general implementation of SMC algorithms.
+#'
+#' RcppSMC can be used to implement SMC algorithms ranging from
+#' simple particle filters to the SMC samplers of Del Moral, Doucet and Jasra (2006) 
+#' within a generic framework. For a general introduction to SMC methods, see
+#' Doucet, Freitas and Gordon (2001) and Doucet and Johansen (2008).
+#'
+#' RcppSMC is based around an Rcpp integration of the SMC template class library (Johansen, 2009).
+#'
+#' At present, three examples are included:
```

Comment:
Is this true any more? Don't you also have your tempered SMC sampler example?

It /is/ a pain keeping top level documentation up to date and we can certainly sanity check everything towards the end, but it always seems worth mentioning even trivial details when spotted.